imports/plugins/core/checkout/client/components/cartItems.js

```diff
@@ -51,10 +51,10 @@ class CartItems extends Component {
         >
           {handleImage(item) ?
             <div className="center-cropped" style={{ backgroundImage: `url('${handleImage(item).url({ store: "small" })}')` }}>
-              <img src={handleImage(item).url({ store: "small" })} className="product-grid-item-images img-responsive" />
+              <img src={handleImage(item).url({ store: "small" })} className="product-grid-item-images img-responsive" alt="" />
             </div> :
             <div className="center-cropped" style={{ backgroundImage: "url('/resources/placeholder.gif')" }}>
-              <img src="/resources/placeholder.gif" className="product-grid-item-images img-responsive" />
+              <img src="/resources/placeholder.gif" className="product-grid-item-images img-responsive" alt="" />
```

Comment:
Should we put something int he `alt` tag, instead of leaving it blank?

Reply:
No, as the image is for decoration. https://www.w3.org/WAI/tutorials/images/decorative/ namely these two bullet points:

- Illustrative of adjacent text but not contributing information (“eye-candy”);
- Identified and described by surrounding text.